components/notice/list.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * External depednencies
+ */
+import { noop } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import Notice from './';
+
+function NoticeList( { notices, onRemove = noop } ) {
+	const removeNotice = ( id ) => () => onRemove( id );
+
+	return (
+		<div className="components-notice-list">
+			{ notices.reverse().map( ( notice ) => (
```

Comment:
@youknowriad: FYI, `Array#reverse` reverses _in place_. I think this has worked so far because `getNotices` returns a new instance every time via `lodash#values` and the prop isn't used elsewhere, but be aware. :)